### PR TITLE
Replace Travis CI for GitHub Actions

### PR DESCRIPTION
**Pull request by karpa4o4**

### CODE REVIEW

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  lint:
+    name: Run lint sctipt
```
Comment (eugulixes) [inline .github/workflows/checks.yml:11]: Probably `Run linter` ?

File: .github/workflows/checks.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        with:
+          node-version: 12
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run lint
+        run: npm run lint
+  test:
+    name: Run test sctipt
```
Comment (eugulixes) [inline .github/workflows/checks.yml:29]: Run tests

File: .github/workflows/checks.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+name: Check sctipt runners
```
Comment (eugulixes) [inline .github/workflows/checks.yml:1]: Tests and Linter Runners

File: .github/workflows/checks.yml (hunk 1 above)
Comment (karpa4o4) [reply .github/workflows/checks.yml:11]: `lint` is the name of the script in the package.json. See [here](https://github.com/tolstoyevsky/cusdeb-web-client/blob/master/package.json#L10).

File: .github/workflows/checks.yml (hunk 2 above)
Comment (karpa4o4) [reply .github/workflows/checks.yml:29]: `test` is the name of the script in the package.json. See [here](https://github.com/tolstoyevsky/cusdeb-web-client/blob/master/package.json#L12).

File: .github/workflows/checks.yml (hunk 1 above)
Comment (karpa4o4) [reply .github/workflows/checks.yml:11]: Okay, let's replace it with the options you suggested.

**Review by eugulixes — APPROVED** [review]

**Pull request merged**